### ISSUE TITLE
The help-text should wrap only after the width of the input box.

### ISSUE THREAD

**Issue by sandeepsajan0** [HyphaApp/hypha#3589]
Help text should utilize the available width.
<img width="795" alt="image" src="https://github.com/HyphaApp/hypha/assets/23638629/bba45a07-4bd0-4ac9-9db9-3722e7100b11">

**Comment by frjo** [comment]
For increased readability it is recommended not to have too long lines of text. Around 70 characters are often quoted as a max.

More that that and many readers struggle to keep track of the next line.

**Comment by theskumar** [comment]
@frjo should we mark this as won't fix.
